### PR TITLE
chore: Upgrade webpack to version 5.

### PR DESCRIPTION
**Pull request by bryans99**
Add analyze scripts to standalone api explorer, extension api explorer and hackathon extension

### CODE REVIEW

File: packages/api-explorer/package.json [hunk 1]
```diff
@@ -72,6 +71,10 @@
     "react-router-dom": "^5.1.2",
     "redux": "^4.0.5",
     "styled-components": "^5.2.1",
-    "ts-jest": "^26.2.0"
+    "ts-jest": "^26.2.0",
+    "webpack-merge": "^5.7.3",
+    "webpack-bundle-analyzer": "^4.2.0",
+    "webpack-cli": "^4.6.0",
+    "webpack-dev-server": "^3.11.2"
```
Comment (jkaster) [inline packages/api-explorer/package.json:78]: can these be `devDependencies`?

File: packages/api-explorer/webpack.dev.config.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
  MIT License
 
- Copyright (c) 2020 Looker Data Sciences, Inc.
+ Copyright (c) 2021 Looker Data Sciences, Inc.
```
Comment (jkaster) [inline packages/api-explorer/webpack.dev.config.js:5]: Must not be lint checked

**Review by jkaster — CHANGES_REQUESTED** [review]
I like the browser config intro. I think the apix-explorer package.json should have some things moved to dev dependencies and that's all I noticed.

File: packages/api-explorer/package.json (hunk 1 above)
Comment (bryans99) [reply packages/api-explorer/package.json:78]: Yes. Dammit I thought I had moved them

File: packages/api-explorer/webpack.dev.config.js (hunk 2 above)
Comment (bryans99) [reply packages/api-explorer/webpack.dev.config.js:5]: I guess so. Need to look at that (in another PR)

File: packages/api-explorer/package.json (hunk 1 above)
Comment (bryans99) [reply packages/api-explorer/package.json:78]: I did move them, but in this package devDependencies come before dependencies. In others they are reversed! Argh!

File: packages/api-explorer/webpack.dev.config.js (hunk 2 above)
Comment (bryans99) [reply packages/api-explorer/webpack.dev.config.js:5]: Actually, they are lint checked. I changed to 2021

Comment (bryans99) [reply packages/api-explorer/webpack.dev.config.js:5]: Just confirmed. `yarn lint-changed-fix` fixed it when I set it back to 2020.

**Review by jkaster — APPROVED** [review]
LGTM. Looking forward to this

File: packages/api-explorer/package.json (hunk 1 above)
Comment (jkaster) [reply packages/api-explorer/package.json]: Caught by inconsistency

**Pull request merged**